ImageInLib/ImageInLib/src/segmentation3d_gsubsurf.c

```diff
@@ -190,12 +235,14 @@ bool generalizedSubsurfSegmentation(Image_Data inputImageData, dataType** segFun
 
 			Storage_Flags storageFlags = { false, false };
 			manageFile(imageData.segmentationFuntionPtr, length, width, height, name, STORE_DATA_RAW, BINARY_DATA, storageFlags);
+			printf("Step is %zd\n", segParameters.numberOfTimeStep);
+			printf("Error = %lf\n", difference_btw_current_and_previous_sol);
 		}
 		i++;
 	} while ((i <= segParameters.maxNoOfTimeSteps) && (difference_btw_current_and_previous_sol > segParameters.segTolerance));
 
-	secondCpuTime = clock() / (dataType)(CLOCKS_PER_SEC);
-	printf("GSUBSURF CPU time: %e secs\n", secondCpuTime - firstCpuTime);
+	//secondCpuTime = clock() / (dataType)(CLOCKS_PER_SEC);
+	//printf("GSUBSURF CPU time: %e secs\n", secondCpuTime - firstCpuTime);
```

Comment:
if you need this output for debugging purposes, please "control it" by defined constants